PaddleCV/PaddleDetection/tools/train.py

```diff
@@ -171,8 +171,9 @@ def main():
             it, np.mean(outs[-1]), logs, end_time - start_time)
         logger.info(strs)
 
-        if it > 0 and it % cfg.snapshot_iter == 0:
-            checkpoint.save(exe, train_prog, os.path.join(save_dir, str(it)))
+        if it > 0 and it % cfg.snapshot_iter == 0 or it == cfg.max_iters - 1:
```

Comment:
why change here

Reply:
Now model_final is not evaluated when training and eval at same time. This change enables this case.